src/models/match.model.ts

```diff
@@ -5,25 +5,25 @@ import { Table, Column, Model, PrimaryKey, DataType } from 'sequelize-typescript
 })
 export default class Match extends Model<Match> {
   @Column
-  bettingLeague: number
+  bettingLeague: string
```

Comment:
Proč je ve většině modelů number vyměněn za string?

Reply:
Jeste se tam museji dodelat vazby mezi tabulkami.